### PR TITLE
Luis plaza unittest

### PR DESCRIPTION
**Pull request by plazluis**

### CODE REVIEW

File: app/src/test/java/com/example/giftly/Font_SizeTests.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        activity.setFontSize(textSize);
+        assertEquals(textSize, textView.getTextSize(), 0);
+        System.out.println("Test passed for testSetFontSize method test.");
+    }
+
+    //Testing whether the save button works
+    @Test
+    public void testSaveButton() {
+        System.out.println("Testing testSaveButton()");
+        Button savingit2 = activity.findViewById(R.id.saveIt2);
```
Comment (itsaruon) [inline app/src/test/java/com/example/giftly/Font_SizeTests.java]: The variable savingit2 is not a meaningful name and may be confusing. Consider using a more descriptive variable name that indicates the purpose of the button being tested.

File: app/src/test/java/com/example/giftly/Font_SizeTests.java [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    @Before
+    public void setUp() {
+        System.out.println("Setting up test...");
+        activity = Robolectric.buildActivity(FontSizeScreen.class).create().visible().get();
+        System.out.println("Setup complete");
+    }
+
+    //Testing on whether the user is able to go into the font size setting
+    @Test
+    public void testOnCreate() {
```
Comment (itsaruon) [inline app/src/test/java/com/example/giftly/Font_SizeTests.java]: Use more descriptive test method names; testOnCreate is generic

**Review by itsaruon — CHANGES_REQUESTED** [review]

File: app/src/test/java/com/example/giftly/Font_SizeTests.java (hunk 1 above)
Comment (plazluis) [reply app/src/test/java/com/example/giftly/Font_SizeTests.java]: renamed the variable to saveButton

File: app/src/test/java/com/example/giftly/Font_SizeTests.java (hunk 2 above)
Comment (plazluis) [reply app/src/test/java/com/example/giftly/Font_SizeTests.java]: renamed the test method name to testAccessFontSizeFeature()

**Review by itsaruon — APPROVED** [review]

**Pull request merged**